model/group_request.go

```diff
@@ -126,6 +120,9 @@ func (p *PatchGroupRequest) Apply(group *Group) bool {
 
 // Validate validates the values of a group patch request
 func (p *PatchGroupRequest) Validate() error {
+	if p.Name != nil && len(*p.Name) == 0 {
```

Comment:
If the value is `nil`, it won't be caught here, right?

Reply:
maybe we need `||`?

Reply:
Thanks for checking on this. I believe this logic is correct after reviewing it. We are basically checking if p.Name is not nil and then if it isn't nil we see if the value is blank `""`. If this line turns out to be true it means the update request is trying to update the name of the group to "", which we should return an error on.